trade-finance-manager-ui/templates/deals/deals.njk

```diff
@@ -3,9 +3,7 @@
 
 {% extends "index.njk" %}
 
-{% block pageTitle %}
-  Trade Finance Manager
-{% endblock %}
+{% block pageTitle %}{{ heading }}{% endblock %}
```

Comment:
Multiline formatting please.

Reply:
Changed all pageTitle blocks to multiline format.